spectrum/xpub.py

```diff
@@ -0,0 +1,65 @@
+from spectrum import logger
+
+
+LOGGER = logger.logger(__name__)
+
+def _log(message, *args, **kwargs):
+    LOGGER.info(message, extra={'app':'elife-xpub'}, *args, **kwargs)
```

Comment:
How is the app property used? Is it meant to just be for the interaction via the page object and not the whole application (i.e. all logging)? (The method could also be replaced by a `functools.partial` wrapper but both are probably equally readable)

Reply:
It somehow feels we have a secondary name that may not belong into this module. Ideally we could just use `LOGGER` which already provides the module name.

Reply:
The browser being used may be looking at different applications (`journal`, `elife-xpub`), so this tells you in which context a button has been found, a form has been filled, etc.

Reply:
> The browser being used may be looking at different applications (`journal`, `elife-xpub`), so this tells you in which context a button has been found, a form has been filled, etc.

Wouldn't the module (`__name__`) already provide the context?

Reply:
I think it would be a good direction to go towards (although there are not enough modules yet for that to be meaningful, will need to extract them first)